holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
The problem with a PR like this is that the description at the top is very brief and nothing worth commenting on. That means to know what the PR is really about I have to look at the code, at which point it becomes very tempting to make comments, even without delving into the exact details i.e just trying to understand what is going on.

That is probably why what Jim suggests makes the most sense - just say you aren't ready to receive comments just yet at the top.